test/GithubApi.Users.test.js

```diff
@@ -0,0 +1,43 @@
+const axios = require('axios');
+const chai = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const { expect } = chai;
+chai.use(require('chai-subset'));
+
+const urlBase = 'https://api.github.com/users';
+const keyPath = axios.create({
+  path: urlBase,
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+
+let params = {
+  per_page: 10
+};
```

Comment:
since this will change in each test, I don't recommend to define a global variable, instead of that, declare and define the variable in each test

Reply:
Would i declare as global just with let and define in each scope?